test/test_cli.py

```diff
@@ -0,0 +1,83 @@
+import shutil
+import pytest
+import subprocess
+import os
+from app import application
+import time
+
+
+@pytest.mark.main
+def test_initialize_repo():
+    try:
+        shutil.rmtree("bitcointranscripts", ignore_errors=True)
+    except Exception as e:
+        assert False
+
+
+@pytest.mark.main
+def test_find_source_type():
+    video = application.check_source_type("B0HW_sJ503Y") == "video"
+
+    playlist = application.check_source_type("PLPQwGV1aLnTuN6kdNWlElfr2tzigB9Nnj") == "playlist"
+
+    audio = application.check_source_type(
+        "https://anchor.fm/s/12fe0620/podcast/play/32260353/https%3A%2F%2Fd3ctxlq1ktw2nl.cloudfront.net%2Fstaging"
```

Comment:
You should use a shorter file. 26 min isn't necessary.

Reply:
I'm in fact scrapping this test as this just checks if pywhisper is working properly or not and I don't think that it needs to be tested.